.zenodo.json

```diff
@@ -0,0 +1,43 @@
+{
+    "creators": [
+        {
+            "orcid": "0000-0001-8313-1644",
+            "name": "Frédéric Chapoton"
+        },
+        {
+            "name": "Diana Davis"
+        },
+        {
+            "orcid": "0000-0002-9608-782X",
+            "name": "Delecroix, Vincent",
+            "affiliation": "CNRS - Université de Bordeaux"
+        },
+        {
+            "name": "Oscar Fontaine"
+        },
+        {
+            "name": "Charles Fougeron"
+        },
+        {
+            "name": "Luke Jeffreys"
+        },
+        {
+            "orcid": "0000-0001-8313-1644",
+            "name": "Samuel Lelièvre",
+            "affiliation": "Université Paris-Saclay"
+        },
+        {
+            "orcid": "0000-0002-3930-9107",
+            "name": "Rüth, Julian"
+        },
+        {
+            "name": "Ivan Yakovlev"
+        },
+        {
+            "name": "Christopher Zhang"
+        }
```

Comment:
@videlec @slel if you happen to know ORCIds or affiliations, feel free to fill them in here.